.jenkins/caffe2/build.sh

```diff
@@ -104,7 +104,7 @@ if [[ "${BUILD_ENVIRONMENT}" == *-android* ]]; then
   build_args+=("BUILD_TEST=ON")
   build_args+=("USE_OBSERVERS=ON")
   build_args+=("USE_ZSTD=ON")
-  "${ROOT_DIR}/scripts/build_android.sh" $(build_to_cmake ${build_args[@]}) "$@"
+  BUILD_CAFFE2_MOBILE=1 "${ROOT_DIR}/scripts/build_android.sh" $(build_to_cmake ${build_args[@]}) "$@"
```

Comment:
Do we still run this CI job? Could find it in .circleci/config.yml

Reply:
It's still used, see `caffe2_linux_build`

Reply:
> caffe2_linux_build

right, but caffe2_linux_build doesn't seem to have any job with `"${BUILD_ENVIRONMENT}" == *-android*` (I meant to ask about the caffe2 android CI)